pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -819,7 +819,7 @@ def get_hamiltonian(self, time: float) -> qutip.Qobj:
     def run(
         self,
         progress_bar: Optional[bool] = False,
-        **options: qutip.solver.Options,
+        **options: dict[str, Any],
     ) -> SimulationResults:
         """Simulates the sequence using QuTiP's solvers.
```

Comment:
Could you please also update the docstring to say that `options` expects the arguments to `qutip.Options()`?

Reply:
Thanks for having a look. I've changed it, and I've also updated the docstring.